test/powershell/Language/Scripting/NativeExecution/NativeUnixGlobbing.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+$PesterSkip = if ( $IsWindows ) { @{ Skip = $true } } else { @{} }
+
+Describe 'Native UNIX globbing tests' -tags "CI" {
+
+    BeforeAll {
+        if (-not $IsWindows )
+        {
+            $tempdir = [System.IO.Path]::GetTempPath() + "/NativeUnixGlobbingTests"
+            /bin/mkdir $tempdir 2> $null
+            "" > "$tempdir/abc.txt"
+            "" > "$tempdir/bbb.txt"
+            "" > "$tempdir/cbb.txt"
+        }
+    }
+
+    AfterAll {
+        if (-not $IsWindows )
+        {
+            /bin/rm -fr $tempdir 2> $null
+        }
+    }
+
```

Comment:
I would rather these tests be done in `$TESTDRIVE` which will eliminate the need for `AfterAll` (`$TESTDRIVE` and contents are automatically deleted by pester)
```
    BeforeAll {
        if (-not $IsWindows )
        {
            "" > "$TESTDRIVE/abc.txt"
            "" > "$TESTDRIVE/bbb.txt"
            "" > "$TESTDRIVE/cbb.txt"
        }
    }
```
and the tests:
```
    # Test simple * expansion
    It 'The globbing pattern should match 3 files' @PesterSkip {
        (/bin/ls $TESTDRIVE/*.txt).Length | Should Be 3
    }
```